src/CommonTerms.js

```diff
@@ -35,6 +35,13 @@ module.exports.DCTERMS = {
   creator: rdf.namedNode(`${DCTERMS_NAMESPACE}creator`),
 };
 
+// For purely legacy reasons (specifically 'cos the core RDF and RDFS vocabs
+// use terms from this 'old' vocab to describe themselves)!
+const DCELEMENTS_NAMESPACE = "http://purl.org/dc/elements/1.1/";
+module.exports.DCELEMENTS = {
+  title: rdf.namedNode(`${DCELEMENTS_NAMESPACE}title`),
```

Comment:
```suggestion
  title: rdf.namedNode(new URL("title", DCELEMENTS_NAMESPACE).href),
```
I tend to use the URL builder everywhere to avoir normalization issues, double slashes and all these. In this case definitely overkill, but it's good practice to avoid using string concat to build IRIs.

Reply:
Yep, sure. Rather than update all the occurrences in this file, I just extended the comment at the top of the file itself to explain this (and give an example like you did here): 954cdb2
Cheers.